spotlight/evaluation.py

```diff
@@ -96,6 +96,60 @@ def sequence_mrr_score(model, test):
     return np.array(mrrs)
 
 
+def precision_recall_at_k(model, test, k):
```

Comment:
Could we make `k` an optional argument (with a default value)?

Reply:
If yes, I guess the only natural default would be `None` and return the P@k/R@k for the whole ranking (although the function name would become a little misleading).

This is why I usually call the function just precision/recall, and put the "at"s only in the variable names. But I guess the current name is what most people expect to find in the library because the literature chose to put the k in the metric name...

Reply:
I think one name that would be consistent with the existing naming convention would be `precision_recall_score`.